README.md

```diff
@@ -8,7 +8,7 @@ The solution provided by Daikin is a mobile app (very well designed) that allows
 
 Even if the system works, there are some problems:
 
-- It is no possible to control the AC from a web browser. You must use the official mobile application. If your device is not supported or you are from a computer you can not control your air conditioner over the internet.
+- It's possible to control the AC from a web browser. You can use the official mobile application also. If your device is not supported or you are from a computer you can not control your air conditioner over the internet.
```

Comment:
I think you misunderstood this section... This is about the official tools provided by Daikin, and as far as I know there is no official web application to control AC units.